src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.cs

```diff
@@ -2126,6 +2140,12 @@ public override BoundNode VisitLambda(BoundLambda node)
             var oldSymbol = this.CurrentSymbol;
             this.CurrentSymbol = node.Symbol;
 
+            // Mark attribute arguments as used.
+            if (node.Symbol.BindMethodAttributes() is { IsDefaultOrEmpty: false } boundAttributes)
```

Comment:
This does binding during flow analysis, which is something @RikkiGibson was concerned about. See https://github.com/dotnet/roslyn/pull/62158#discussion_r917129834

Reply:
I see, although this was accepted for [default lambda parameters](https://github.com/dotnet/roslyn/pull/63850/files#diff-c1fe10ac125bdb06ceee8d34353277788c8751b28a729cd3d5ce55e8deb8c91bR1690) and [parameter attributes](https://github.com/dotnet/roslyn/pull/64660/files#diff-c1fe10ac125bdb06ceee8d34353277788c8751b28a729cd3d5ce55e8deb8c91bR1690).

Reply:
Yeah, I think at this point we agree that as long as it doesn't present a performance issue, it is not a serious problem to re-bind something which may already be bound somewhere else. Thanks!